test/absorb-test.ts

```diff
@@ -355,14 +356,15 @@ describe('absorb', function () {
     const { comet, tokens, users: [absorber, underwater], priceFeeds } = protocol;
     const { COMP, WBTC, WETH } = tokens;
 
-    const debt = 1n - (exp(41000, 6) + exp(3000, 6) + exp(175, 6));
+    const finalDebt = 1n;
+    const startingDebt = finalDebt - (exp(41000, 6) + exp(3000, 6) + exp(175, 6));
```

Comment:
what are the 41K, 3K and 175 values here?

Reply:
These are from Kevin's branch, he updated this test to be more logical while he was at it - I believe those are coming from the prices?